File: Python/LAB3_arreglos/Fecha.py
class Fecha:
    def __init__(self,dd,mm,aa):
        self.__dd = dd
        self.__mm = mm
        self.__aa = aa

    def getDia(self):
        return self.__dd
    
    def setDia(self,dd):
        self.__dd = dd

    def getMes(self):
        return self.__mm
    
    def setMes(self,mm):
        self.__mm = mm

    def getA(self):
        return self.__aa
    
    def setA(self,aa):
        self.__aa = aa
    
    def __str__(self):
        return (f'{self.__dd}-{self.__mm}-{self.__aa}')

# fecha = Fecha("25","Mar","2023")
# print(fecha.getDia())fecha = Fecha("25","Mar","2023")
# print(fecha.getDia())

File: Java/LAB3_arreglos/Lab3/JavaApplication54/src/Lab3/Fecha.java


package Lab3;


public class Fecha {
    //Atributos
    private short dd;
    private short mm;
    private short aa;
    
    //Constructores

    public Fecha() {
    }

    public Fecha(short dd, short mm, short aa) {
        this.dd = dd;
        this.mm = mm;
        this.aa = aa;
    }
    //Getter and setters
    public short getDia() {
        return dd;
    }

    public void setDia(short dd) {
        this.dd = dd;
    }

    public short getMes() {
        return mm;
    }

    public void setMes(short mm) {
        this.mm = mm;
    }

    public short getA() {
        return aa;
    }

    public void setA(short aa) {
        this.aa = aa;
    }

    @Override
    public String toString() {
        return (dd+"-"+ mm +"-"+aa);
    }
    
}
    
    
    
    
    